emperor/support_files/js/color-view-controller.js

```diff
@@ -258,6 +258,31 @@ define([
   };
 
   /**
+   * Converts the current instance into a JSON string.
+   *
+   * @return {Object} JSON ready representation of self.
+   */
+  ColorViewController.prototype.toJSON = function() {
+    var json = EmperorAttributeABC.prototype.toJSON.call(this);
+    json.colormap = this.$colormapSelect.val();
+    json.continuous = this.$scaled.is(':checked');
+    return json;
+  }
+
+  /**
+   * Decodes JSON string and modifies its own instance variables accordingly.
+   *
+   * @param {Object} Parsed JSON string representation of self.
+   */
+  ColorViewController.prototype.fromJSON = function(json) {
+    this.$colormapSelect.val(json.colormap);
```

Comment:
Would you mind adding a small comment describing why the order of these lines is important?